content/01.abstract.md

```diff
@@ -1,3 +1,7 @@
 ## Abstract {.page_break_before}
 
-
+Scientific literature reviews are ideal for open, collaborative writing.
+Allowing any interested individual to contribute as an author can strengthen a review, providing broad and fair coverage of the subject matter.
+However, the traditional multi-author writing process breaks down at scale.
+We present techniques for overcoming the challenges of open manuscript writing.
+These include approaches for managing distributed authors and our new software for automating citation and manuscript building.
```

Comment:
I think the abstract needs to mention Manubot by name and provide some more information on it. One point after all is for this article to be the publication citation for the Manubot. I'm happy to propose some text here... probably once we merge this PR (the content that is here is good and mergeable).

Reply:
Sounds good.  I'll merge this so you can take a pass over the abstract.